annotators/bot_emotion_classifier/service_configs/bot-emotion-classifier/service.yml

```diff
@@ -0,0 +1,28 @@
+name: bot-emotion-classifier
+endpoints:
+- model
```

Comment:
endpoints are different in your server.py

Reply:
seems correct to me https://github.com/deeppavlov/dream/blob/2fda668ba6584aeffd598e4cc9ce838cce2419ed/annotators/bot_emotion_classifier/server.py#L349

Reply:
my bad